pvlib/test/test_scaling.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pandas as pd
+
+import pytest
+from numpy.testing import assert_almost_equal
+
+from pvlib import scaling
+from conftest import requires_scipy
+
+
+# Sample positions
+lat = np.array((9.99, 10, 10.01))
+lon = np.array((4.99, 5, 5.01))
+coordinates = np.array([(lati, loni) for (lati, loni) in zip(lat, lon)])
+
+# Sample cloud speed
+cloud_speed = 5
+# Generate a sample clear_sky_index and time vector.
+clear_sky_index = np.ones(10000)
+clear_sky_index[5000:5005] = np.array([1, 1, 1.1, 0.9, 1])
+time = np.arange(0, len(clear_sky_index))
+
+# Sample dt
+dt = 1
+
+# Expected positions for sample lat/lon given above (calculated manually)
+expect_xpos = np.array([554863.4, 555975.4, 557087.3])
+expect_ypos = np.array([1110838.8, 1111950.8, 1113062.7])
```

Comment:
maybe move these into the proposed `positions` fixture below and use indexing to get a single coordinate if needed.